src/utils/search/search.service.ts

```diff
@@ -9,6 +9,7 @@ import { SearchResultEntry } from './search-result-entry.dto';
 import { ISearchResultEntry } from './search-result-entry.interface';
 import { LogContext } from '../logging/logging.contexts';
 import { ValidationException } from '../error-handling/exceptions/validation.exception';
+import { group } from 'console';
```

Comment:
wrong import?

Reply:
Not sure where that came from, will remove it.